src/LayerEditor/ui/diagram_editor/diagram_scene.py

```diff
@@ -18,22 +18,19 @@ class DiagramScene(QtWidgets.QGraphicsScene):
     # selection has changed
 
     regionsUpdateRequired = QtCore.pyqtSignal()
+    TOLERANCE = 0.01
 
-    def __init__(self, regions, block, parent=None):
-        rect = QtCore.QRectF(-622500, 1128600, 400, 500)
+    def __init__(self, block, parent):
+        rect = QtCore.QRectF(QPoint(-100000, -100000), QPoint(100000, 100000))
```

Comment:
The default rectangle was selected as a rectangle around the origin of JTSK coordinate system used dominantly in the Czech republic. One idea was to have the bounding rectangle of CR as the default `rect` but it could only be meaningful
once we can provide a simple map as the background.  

Also notice that Python constructor takes top-left corner and size.

Reply:
This is only temporary solution, this should be set by initialization area. The original default rect made viewing geometries from original LE impossible because they were outside of it.

Reply:
Use supplement init area from the input file.

Reply:
Supplement is now used for both zoom and position initialization + for setting sceneRect